Samples/excel-insert-file/README.md

```diff
@@ -0,0 +1,177 @@
+---
+page_type: sample
+products:
+- office-excel
+- office
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: "5/18/2022 10:00:00 AM"
+description: "This sample shows how to insert a template from an external Excel file and populate it with JSON data."
+---
+
+# Insert an external Excel file and populate it with JSON data
+
+## Summary
+
+This sample shows how to insert an existing template from an external Excel file into the currently open Excel file. Then it retrieves data from a JSON web service and populates the template for the customer.
+
+> **Note:** The features used in this sample are currently in preview and subject to change. They are not currently supported for use in production environments. To try the preview features, you'll need to [join Office Insider](https://insider.office.com/join). A good way to try out preview features is to sign up for an Office 365 subscription. If you don't already have an Office 365 subscription, get one by joining the [Office 365 Developer Program](https://developer.microsoft.com/office/dev-program).
+
+## Features
+
+- Use **insertWorksheetsFromBase64** to insert a worksheet from another Excel file into the open Excel file.
+- Get JSON data and add it to the worksheet.
+
+## Applies to
+
+- Excel on Windows, Mac, and on the web.
+
+## Prerequisites
+
+To use this sample, you'll need to [join Office Insider](https://insider.office.com/join).
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Insert an external Excel file and populate it with JSON data | Microsoft
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0 | 5-18-2021 | Initial release
+
+----------
+
+## Run the sample
+
+To run the sample you just need to sideload the manifest. The add-in web files are served from this repo on GitHub.
+
+1. Download the **manifest.xml** and **SalesTemplate.xlsx** files from this sample to a folder on your computer.
+1. Open [Office on the web](https://office.live.com/).
+1. Choose **Excel**, and then open a new document.
+1. Open the **Insert** tab on the ribbon and choose **Office Add-ins**.
+1. On the **Office Add-ins** dialog, select the **MY ADD-INS** tab, choose **Manage My Add-ins**, and then **Upload My Add-in**.
```

Comment:
```suggestion
1. On the **Office Add-ins** dialog, select **MY ADD-INS** , tap the **Manage My Add-ins** drop-down, and then **Upload My Add-in**.
```

Reply:
Based on Linda's answer in a different comment on this PR, I recommend "choose" instead of "tap": 
```suggestion
1. On the **Office Add-ins** dialog, select **MY ADD-INS** , choose the **Manage My Add-ins** drop-down, and then choose **Upload My Add-in**.
```

Reply:
For sure, have NO IDEA how/why I suggested that. <<<shaking head, rolling eyeballs>>